src/Quasar/Error.purs

```diff
@@ -64,3 +67,21 @@ type QResponse resp = Either QError resp
 type QContinuation resp next = QResponse resp → next
 
 infixr 2 type QContinuation as :~>
+
+data PDFError = NoCEFPathInConfig | CEFPDFError String
+
+instance showPDFError ∷ Show PDFError where
+  show = case _ of
+    NoCEFPathInConfig →
+      "NoCEFPathInConfig"
+    CEFPDFError string →
+      "(CEFPDFError " <> string <> ")"
+
+printPDFError ∷ PDFError → String
+printPDFError = case _ of
+  NoCEFPathInConfig →
+    "The PDF service is unavailable, there is no path to the PDF service in the SlamData configuration file"
+  CEFPDFError "command not found" →
```

Comment:
Perhaps this should be a constructor if we're matching it for a specific error anyway?

Reply:
Oh, yeah that sounds like it would be worth to be added as a case for PDFError and checked during decoding rather than during printing.

Reply:
I initially added a constructor but that made me realise that

a) thats not really matching what Quasar gives us
b) different operating systems and different localisations may return different strings

so i thought it would be preferable to remove this special case.